packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.stories.tsx

```diff
@@ -0,0 +1,402 @@
+import {html} from 'lit-html';
+import defaultStory from '../../../.storybook/default-story';
+
+const {defaultModuleExport, exportedStory} = defaultStory(
+  'Atomic/ResultList',
```

Comment:
This should be `Atomic/FoldedResultList` or there will be a "collision" in the page ID that gets generated.